# 365 Probabilidades — Dia #018
## Qual a probabilidade de o seu senso de humor estar te protegendo da morte?

**Tipo:** Preditivo  
**Data de publicação:** 2026-07-01  
**Ferramenta:** Python  
**Decisão analisada:** Levar a vida com bom humor é só um jeito de ser, ou tem impacto real na saúde?  
**Hashtag:** #365Probabilidades #Dia018

---

### 📖 A História

Existe um tipo de pessoa que todo mundo reconhece: aquela que encontra uma piada até na fila do banco, que ri da própria desgraça, que transforma qualquer situação tensa em algo mais leve. Geralmente isso é tratado como traço de personalidade — engraçado, mas irrelevante para questões mais "sérias" da vida, como saúde e longevidade.

Pesquisadores noruegueses discordam, com 15 anos de dados para sustentar isso.

Eles acompanharam mais de 53 mil pessoas durante 15 anos, medindo o senso de humor de cada uma no início do estudo e observando, ao longo do tempo, quem sobrevivia e por que motivo as pessoas morriam. O resultado: quem tinha senso de humor mais alto morria significativamente menos — e a diferença era especialmente forte para um tipo específico de causa de morte.

---

### 📚 O Conceito

O estudo, parte do Nord-Trøndelag Health Study (HUNT) na Noruega, usou o Sense of Humor Questionnaire (SHQ) para medir três componentes do senso de humor: cognitivo (capacidade de perceber o humor em situações), social (uso do humor em interações) e afetivo (apreciação genuína do humor).

A pesquisa testou hipóteses específicas sobre diferentes causas de morte — doenças cardiovasculares, infecções, câncer e doença pulmonar obstrutiva crônica — em vez de olhar apenas para mortalidade geral, o que é mais raro e mais informativo na literatura sobre humor e saúde.

---

### 🧮 O Modelo
**Fontes:**
- Romundstad, S., Svebak, S., Holen, A. & Holmen, J. (2016) — *A 15-Year Follow-Up Study of Sense of Humor and Causes of Mortality: The Nord-Trøndelag Health Study* — Psychosomatic Medicine, 78(3) — N=53.556
- Svebak, S., Romundstad, S. & Holmen, J. (2010) — *A 7-Year Prospective Study of Sense of Humor and Mortality in an Adult County Population: The HUNT-2 Study* — International Journal of Psychiatry in Medicine, 40(2) — N=66.140 participantes (item cognitivo do SHQ avaliado em N=53.546)

In [5]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print('✅ Bibliotecas carregadas')

✅ Bibliotecas carregadas


In [6]:
# --- DADOS DA LITERATURA ---
# Romundstad et al. 2016 — Psychosomatic Medicine — N=53.556, 15 anos de acompanhamento
# Hazard Ratios (HR) — valores abaixo de 1.0 indicam REDUÇÃO de risco

n_estudo = 53556

# Mortalidade geral (all-cause)
hr_mortalidade_geral_mulheres = 0.52   # IC 95%: [0.33, 0.81]
hr_mortalidade_geral_homens   = 0.88   # IC 95%: [0.59, 1.32] — não significativo

# Mortalidade por doença cardiovascular (CVD)
hr_cvd_mulheres = 0.27   # IC 95%: [0.15, 0.47]

# Mortalidade por infecção
hr_infeccao_homens   = 0.26   # IC 95%: [0.09, 0.74]
hr_infeccao_mulheres = 0.17   # IC 95%: [0.04, 0.76]

fator_correcao = 0.80

print('=' * 65)
print('  DADOS DA LITERATURA — SENSO DE HUMOR E MORTALIDADE')
print('=' * 65)
print(f'\n  Romundstad et al. 2016 (N={n_estudo:,}, 15 anos):')
print(f'  → Mortalidade geral — mulheres : HR={hr_mortalidade_geral_mulheres} (significativo)')
print(f'  → Mortalidade geral — homens   : HR={hr_mortalidade_geral_homens} (não significativo)')
print(f'  → Mortalidade CVD — mulheres   : HR={hr_cvd_mulheres}')
print(f'  → Mortalidade infecção — homens  : HR={hr_infeccao_homens}')
print(f'  → Mortalidade infecção — mulheres: HR={hr_infeccao_mulheres}')
print(f'\n  Fator de correção aplicado: x{fator_correcao}')
print('=' * 65)

  DADOS DA LITERATURA — SENSO DE HUMOR E MORTALIDADE

  Romundstad et al. 2016 (N=53,556, 15 anos):
  → Mortalidade geral — mulheres : HR=0.52 (significativo)
  → Mortalidade geral — homens   : HR=0.88 (não significativo)
  → Mortalidade CVD — mulheres   : HR=0.27
  → Mortalidade infecção — homens  : HR=0.26
  → Mortalidade infecção — mulheres: HR=0.17

  Fator de correção aplicado: x0.8


In [7]:
# --- O MODELO ---
# Conversao de Hazard Ratio para redução percentual de risco: (1 - HR) * 100

reducao_mortalidade_geral_mulheres = (1 - hr_mortalidade_geral_mulheres) * 100
reducao_cvd_mulheres               = (1 - hr_cvd_mulheres) * 100
reducao_infeccao_homens            = (1 - hr_infeccao_homens) * 100
reducao_infeccao_mulheres          = (1 - hr_infeccao_mulheres) * 100

# Aplicando fator de correção sobre as reduções centrais
reducao_geral_corrigida    = reducao_mortalidade_geral_mulheres * fator_correcao
reducao_cvd_corrigida      = reducao_cvd_mulheres * fator_correcao
reducao_infeccao_h_corrig  = reducao_infeccao_homens * fator_correcao
reducao_infeccao_m_corrig  = reducao_infeccao_mulheres * fator_correcao

print('=' * 65)
print('  MODELO — RESULTADO (Redução de risco com alto senso de humor)')
print('=' * 65)
print(f'  Mortalidade geral (mulheres):')
print(f'  → Redução de risco : {reducao_mortalidade_geral_mulheres:.0f}%')
print(f'  → Corrigido         : {reducao_geral_corrigida:.0f}%')
print(f'\n  Mortalidade por doença cardiovascular (mulheres):')
print(f'  → Redução de risco : {reducao_cvd_mulheres:.0f}%')
print(f'  → Corrigido         : {reducao_cvd_corrigida:.0f}%')
print(f'\n  Mortalidade por infecção (homens):')
print(f'  → Redução de risco : {reducao_infeccao_homens:.0f}%')
print(f'  → Corrigido         : {reducao_infeccao_h_corrig:.0f}%')
print(f'\n  Mortalidade por infecção (mulheres):')
print(f'  → Redução de risco : {reducao_infeccao_mulheres:.0f}%')
print(f'  → Corrigido         : {reducao_infeccao_m_corrig:.0f}%')
print('=' * 65)

  MODELO — RESULTADO (Redução de risco com alto senso de humor)
  Mortalidade geral (mulheres):
  → Redução de risco : 48%
  → Corrigido         : 38%

  Mortalidade por doença cardiovascular (mulheres):
  → Redução de risco : 73%
  → Corrigido         : 58%

  Mortalidade por infecção (homens):
  → Redução de risco : 74%
  → Corrigido         : 59%

  Mortalidade por infecção (mulheres):
  → Redução de risco : 83%
  → Corrigido         : 66%


In [8]:
# --- VISUALIZACAO ---

# GRAFICO 1 — Diferença de mortalidade por causa e gênero
fig1, ax1 = plt.subplots(figsize=(12, 8))

categorias = ['Mortalidade\ngeral\n(mulheres)', 'Mortalidade\nCVD\n(mulheres)',
              'Mortalidade\ninfecção\n(homens)', 'Mortalidade\ninfecção\n(mulheres)']
reducoes = [reducao_mortalidade_geral_mulheres, reducao_cvd_mulheres,
            reducao_infeccao_homens, reducao_infeccao_mulheres]
cores1 = ['#2a8a82', '#c8a84b', '#c0392b', '#c0392b']

bars = ax1.bar(categorias, reducoes, color=cores1, alpha=0.85, width=0.55)
ax1.set_ylim(0, 95)
ax1.set_ylabel('Diferença observada na mortalidade com alto senso de humor (%)', fontsize=12)
ax1.set_title('Senso de humor associado a menor mortalidade por causas específicas\nRomundstad et al. (2016) — N=53.556, 15 anos de acompanhamento',
              fontsize=13, pad=15)

for bar, r in zip(bars, reducoes):
    ax1.text(bar.get_x() + bar.get_width() / 2, r + 2,
             f'-{r:.0f}%', ha='center', fontweight='bold', fontsize=14)

plt.tight_layout()
plt.savefig('dia-018-grafico-01-reducao-risco.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 1 salvo!')


# GRAFICO 2 — Comparação Hazard Ratio (humor) vs fatores de risco clássicos
fig2, ax2 = plt.subplots(figsize=(12, 8))

fatores = ['Senso de humor\nalto (mulheres)', 'Diabetes', 'Câncer', 'Doença\ncardiovascular']
hrs = [hr_mortalidade_geral_mulheres, 4.86, 4.18, 6.28]
cores2 = ['#2a8a82', '#c8a84b', '#c0392b', '#c0392b']

bars2 = ax2.bar(fatores, hrs, color=cores2, alpha=0.85, width=0.55)
ax2.axhline(y=1.0, color='#333333', linestyle='--', linewidth=1.5, label='HR=1,0 (sem efeito)')
ax2.set_ylabel('Hazard Ratio (HR)', fontsize=12)
ax2.set_title('Senso de humor na direção oposta dos fatores clássicos\nHumor: Romundstad et al. 2016 (N=53.556) · Fatores clássicos: Svebak et al. 2010 (N=66.140)',
              fontsize=13, pad=15)
ax2.legend(fontsize=11)

for bar, hr in zip(bars2, hrs):
    ax2.text(bar.get_x() + bar.get_width() / 2, hr + 0.15,
             f'HR={hr}', ha='center', fontweight='bold', fontsize=12)

plt.tight_layout()
plt.savefig('dia-018-grafico-02-comparacao-fatores.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 2 salvo!')


# GRAFICO 3 — Distribuição da associação (mortalidade infecção, mulheres)
fig3, ax3 = plt.subplots(figsize=(12, 8))

# Usando IC 95% reportado diretamente no estudo: [0.04, 0.76]
ic_inf_mulheres = (0.04, 0.76)
x = np.linspace(0.01, 1.0, 1000)

# Aproximação visual da distribuição do HR via log-normal (metodologia padrão para HR)
log_hr = np.log(hr_infeccao_mulheres)
log_ic_low = np.log(ic_inf_mulheres[0])
log_ic_high = np.log(ic_inf_mulheres[1])
se_log_hr = (log_ic_high - log_ic_low) / (2 * 1.96)

y = stats.norm.pdf(np.log(x), loc=log_hr, scale=se_log_hr) / x

ax3.plot(x, y, color='#c0392b', linewidth=3)
ax3.fill_between(x, y, alpha=0.2, color='#c0392b')
ax3.axvline(x=ic_inf_mulheres[0], color='#c8a84b', linestyle='--', linewidth=2,
            label=f'IC 95%: [{ic_inf_mulheres[0]:.2f}, {ic_inf_mulheres[1]:.2f}]')
ax3.axvline(x=ic_inf_mulheres[1], color='#c8a84b', linestyle='--', linewidth=2)
ax3.axvline(x=hr_infeccao_mulheres, color='#c0392b', linewidth=2,
            label=f'HR central: {hr_infeccao_mulheres}')
ax3.axvline(x=1.0, color='#333333', linestyle=':', linewidth=1.5, label='HR=1,0 (sem efeito)')

ax3.set_xlabel('Hazard Ratio — Mortalidade por infecção (mulheres)', fontsize=12)
ax3.set_ylabel('Densidade', fontsize=12)
ax3.set_title('Distribuição da associação entre humor e mortalidade por infecção\nRomundstad et al. (2016) — N=53.556',
              fontsize=13, pad=15)
ax3.legend(fontsize=11)
ax3.set_xlim(0, 1.2)

plt.tight_layout()
plt.savefig('dia-018-grafico-03-bayesiano.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 3 salvo!')

✅ Grafico 1 salvo!
✅ Grafico 2 salvo!
✅ Grafico 3 salvo!


### 💡 O Insight

Em estudo com **53.556 pessoas seguidas por 15 anos**, mulheres com alto senso de humor (componente cognitivo) apresentaram **48% menos mortalidade por qualquer causa** (HR=0,52) do que mulheres com baixo senso de humor. A diferença foi ainda maior para causas específicas: **73% menos mortalidade por doença cardiovascular** em mulheres, e **74% a 83% menos mortalidade por infecção** — em homens e mulheres, respectivamente.

Para comparação: o mesmo banco de dados mostrou que diabetes está associada a um aumento de risco de HR=4,86, e câncer a HR=4,18. O senso de humor caminha na direção oposta, com magnitude comparável — só que na direção de menor mortalidade observada, não maior.

Isso não significa que rir mais vai necessariamente alongar sua vida. É um estudo observacional: pessoas com mais senso de humor também podem ter, em média, menos estresse crônico, mais engajamento social ou melhor saúde mental — fatores que podem explicar parte da associação independentemente do humor em si. O que os dados permitem dizer, com confiança, é que existe uma associação real e mensurável entre o componente cognitivo do senso de humor e menor mortalidade nessa população — não que um cause o outro.

*Quando foi a última vez que você riu de verdade — e o que isso pode estar dizendo sobre o seu corpo, além da sua cabeça?*

---

### ⚠️ Limitações do Modelo
- O estudo é observacional — não é possível afirmar causalidade direta. É plausível, inclusive, que a causalidade seja reversa: doença subclínica (ainda não diagnosticada) pode reduzir o senso de humor anos antes da morte, em vez de o humor causar a sobrevivência
- O efeito foi estatisticamente significativo em mulheres para mortalidade geral, mas não em homens — a associação por causa específica de morte variou por gênero
- Dos três componentes do SHQ (cognitivo, social, afetivo), apenas o componente cognitivo mostrou associação significativa com mortalidade — os resultados deste modelo se referem especificamente a essa dimensão, não ao "senso de humor" como conceito geral
- No estudo de 2010 (HUNT-2), a associação entre humor e sobrevivência desaparecia após os 65 anos — o efeito parece concentrado na população mais jovem da amostra
- Foram testadas múltiplas combinações (3 componentes de humor × várias causas de morte × 2 sexos) — quanto mais comparações estatísticas, maior a chance de algum resultado aparecer significativo por acaso
- A amostra é de uma única região da Noruega — generalização para outros contextos culturais requer cautela
- O fator de correção (×0,80) é uma aproximação padrão do projeto

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*